appdatabase/migrations/sql/1690734500_move_wallet_tables_to_wallet_db.down.sql

```diff
@@ -0,0 +1,160 @@
+blk_number BIGINT NOT NULL,
+blk_hash BIGINT NOT NULL,
+loaded BOOL DEFAULT FALSE,
+CONSTRAINT unique_mapping_for_account_to_block_per_network UNIQUE (address,blk_hash,network_id)
+);
+
+CREATE TABLE blocks_ranges (
+network_id UNSIGNED BIGINT NOT NULL,
+address VARCHAR NOT NULL,
+blk_from BIGINT NOT NULL,
+blk_to BIGINT NOT NULL
+, balance BLOB, nonce INTEGER);
+
+CREATE TABLE blocks_ranges_sequential (
+    network_id UNSIGNED BIGINT NOT NULL,
+    address VARCHAR NOT NULL,
+    blk_start BIGINT,
+    blk_first BIGINT NOT NULL,
+    blk_last BIGINT NOT NULL,
+    PRIMARY KEY (network_id, address)
+) WITHOUT ROWID;
+
+CREATE TABLE pending_transactions (
+  network_id UNSIGNED BIGINT NOT NULL,
+  hash VARCHAR NOT NULL,
+  timestamp UNSIGNED BIGINT NOT NULL,
+  from_address VARCHAR NOT NULL,
+  to_address VARCHAR,
+  symbol VARCHAR,
+  gas_price BLOB,
+  gas_limit BLOB,
+  value BLOB,
+  data TEXT,
+  type VARCHAR,
+  additional_data TEXT, multi_transaction_id INT,
+  PRIMARY KEY (network_id, hash)
+) WITHOUT ROWID;
+
+CREATE TABLE "saved_addresses" (
+  address VARCHAR NOT NULL,
+  name TEXT NOT NULL,
+  favourite BOOLEAN NOT NULL DEFAULT FALSE,
+  removed BOOLEAN NOT NULL DEFAULT FALSE,
+  update_clock INT NOT NULL DEFAULT 0,
+  chain_short_names VARCHAR DEFAULT "",
+  ens_name VARCHAR DEFAULT "",
+  is_test BOOLEAN DEFAULT FALSE, created_at INT DEFAULT 0,
+  PRIMARY KEY (address, ens_name, is_test)
+) WITHOUT ROWID;
+
+-- token_balances is the only table that was placed by mistake in nodeconfig
+-- migrations so to avoid possible issues, use NOT EXISTS
+CREATE TABLE IF NOT EXISTS token_balances (
+  user_address VARCHAR NOT NULL,
+  token_name VARCHAR NOT NULL,
+  token_symbol VARCHAR NOT NULL,
+  token_address VARCHAR NOT NULL,
+  token_color VARCHAR NOT NULL DEFAULT "",
+  token_decimals INT NOT NULL,
+  token_description VARCHAR NOT NULL DEFAULT "",
+  token_url VARCHAR NOT NULL DEFAULT "",
+  balance VARCHAR NOT NULL,
+  chain_id INT NOT NULL,
+  PRIMARY KEY (user_address, chain_id, token_symbol) ON CONFLICT REPLACE
+);
+
+CREATE TABLE tokens (
+  address VARCHAR NOT NULL,
+  network_id UNSIGNED BIGINT NOT NULL,
+  name TEXT NOT NULL,
+  symbol VARCHAR NOT NULL,
+  decimals UNSIGNED INT,
+  color VARCHAR,
+  PRIMARY KEY (address, network_id)
+) WITHOUT ROWID;
+
+CREATE TABLE visible_tokens (
+  chain_id UNSIGNED INT,
+  address VARCHAR NOT NULL
+);
+
+CREATE TABLE currency_format_cache (
+    symbol VARCHAR NOT NULL,
+    display_decimals INT NOT NULL,
+    strip_trailing_zeroes BOOLEAN NOT NULL
+);
+
+CREATE TABLE multi_transactions (
+    from_address VARCHAR NOT NULL,
+    from_asset VARCHAR NOT NULL,
+    from_amount VARCHAR NOT NULL,
+    to_address VARCHAR NOT NULL,
+    to_asset VARCHAR NOT NULL,
+    type VARCHAR NOT NULL,
+    timestamp UNSIGNED BIGINT NOT NULL
+, to_amount VARCHAR, from_network_id UNSIGNED BIGINT, to_network_id UNSIGNED BIGINT, cross_tx_id VARCHAR DEFAULT "", from_tx_hash BLOB, to_tx_hash BLOB);
```

Comment:
not sure if this is github misbehaving, seeing weird formatting here
![image](https://github.com/status-im/status-go/assets/11161531/ba566f64-7372-45d1-8095-7ca2b41d1d79)